frontend/app/webroot/js/templates/pages/about.ejs

```diff
@@ -4,7 +4,7 @@
 </div>
 <div class="modal-body">
 	<p>
-		CitePlag is a prototype of a hybrid Plagiarism Prevention and Detection System developed by the <a href="http://www.isg.uni-konstanz.de" target= "_blank">Information Science Group at the University of Konstanz</a>. CitePlag implements the <a href="http://www.isg.uni-konstanz.de/projects/citeplag/" target= "_blank">Citation-based Plagiarism Detection (CbPD)</a> approach, which was initially introduced in the <a href="http://www.gipp.com/wp-content/papercite-data/pdf/thesisbelagipp.pdf">doctoral thesis</a> of <a href="http://www.isg.uni-konstanz.de/people/bela-gipp/" target="_blank">Bela Gipp</a>. Details on the algorithms implemented in <a href="http://gipp.com/wp-content/papercite-data/pdf/gipp11c.pdf">CitePlag can be found here</a>. The current prototype was developed in cooperation with students from the <a href="http://www.htw-berlin.de/" target= "_blank">HTW, Berlin</a>
+		CitePlag is a prototype of a hybrid Plagiarism Prevention and Detection System developed by the <a href="http://www.gipplanb.org" target= "_blank">Chair for Scientific Information Analytics, University of Göttingen</a>. CitePlag implements the <a href="https://gipplab.org/projects/hyplag/" target= "_blank">Citation-based Plagiarism Detection (CbPD)</a> approach, which was initially introduced in the <a href="http://www.gipp.com/wp-content/papercite-data/pdf/thesisbelagipp.pdf">doctoral thesis</a> of <a href="https://gipplab.org/team/prof-dr-bela-gipp/" target="_blank">Bela Gipp</a>. Details on the algorithms implemented in <a href="http://gipp.com/wp-content/papercite-data/pdf/gipp11c.pdf">CitePlag can be found here</a>. The current prototype was developed in cooperation with students from the <a href="http://www.htw-berlin.de/" target= "_blank">HTW, Berlin</a>
```

Comment:
Please replace the link to the thesis with: https://gipplab.org/wp-content/papercite-data/pdf/thesisbelagipp.pdf